libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2523,8 +2523,10 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 	TypePointer exprType = type(_memberAccess.expression());
 	ASTString const& memberName = _memberAccess.memberName();
 
+	auto& annotation = _memberAccess.annotation();
```

Comment:
I think it's butter to call the function twice instead of adding another variable.

Reply:
that may be, but I didn't add another one, this is just moved up from line 2543

Reply:
in this case to take advantage of the existing variable even earlier. ( line 2529)

Reply:
Ah right! I still think it's unnecessary, but you can also keep it.